.yarn/sdks/typescript/lib/tsserver.js

```diff
@@ -34,26 +37,39 @@ const moduleWrapper = tsserver => {
       // with peer dep (otherwise jumping into react-dom would show resolution
       // errors on react).
       //
-      const resolved = pnpApi.resolveVirtual(str);
+      const resolved = normalize(isVirtual(str) ? pnpApi.resolveVirtual(str) : str);
       if (resolved) {
         const locator = pnpApi.findPackageLocator(resolved);
         if (locator && dependencyTreeRoots.has(`${locator.name}@${locator.reference}`)) {
-         str = resolved;
+          str = resolved;
         }
       }
 
-      str = str.replace(/\\/g, `/`)
-      str = str.replace(/^\/?/, `/`);
```

Comment:
These still need to happen for `default` and `vscode`

Reply:
yep I just fixed it